### PR TITLE
feat(app): restore account routes, updates and logout

### PR DESCRIPTION
**Pull request by radiansz**

### CODE REVIEW

File: packages/sendsay-legacy/pages/account/updates/index.js [hunk 1]
```diff
@@ -20,16 +20,16 @@ define(function(require) {
...
-                lastRead = storage.get('data.lastRead');
-            var curLast = this.updatesContent[0].date;
-            if(!lastRead || curLast != lastRead) {
-                if(!storage.get('data'))
-                    storage.set('data', {});
-                storage.get('data').lastRead = curLast;
-                storage.save( { silent:true } );
-            }
+            var page = this;
+            //     storage = this.models.storage,
```
Comment (zouxuoz) [inline packages/sendsay-legacy/pages/account/updates/index.js]: remove it?

File: src/routes/account/components/AccountNavigation.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  settings: {
+    id: 'account-navigation__settings',
+    defaultMessage: 'Настройки',
+  },
+});
+
+const AccountNavigation = ({ intl }) => (
+  <SectionNavigation>
+    <SectionNavigationLink to="/account/billing" label={intl.formatMessage(messages.pay)} />
+    <SectionNavigationLink to="/account/pricing" label={intl.formatMessage(messages.rates)} />
```
Comment (zouxuoz) [inline src/routes/account/components/AccountNavigation.js]: pricing should be first link

**Comment by zouxuoz** [comment]
@radiansz images in updates doesn't work

**Comment by radiansz** [comment]
Issue with images will be fixed in #429

**Comment by zouxuoz** [comment]
@radiansz test comment

File: src/routes/changelog/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ChangelogHeader from './containers/ChangelogHeaderContainer';
+import Changelog from './containers/ChangelogContainer';
```
Comment (rosendi) [inline src/routes/changelog/index.js]: ChangelogContainer

File: src/routes/logout/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  componentWillMount() {
+    const { logout: performLogout } = this.props;
+
+    performLogout();
+  }
+
+  render() {
+    return (
```
Comment (rosendi) [inline src/routes/logout/index.js]: remove the parentheses

File: src/routes/changelog/index.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+const ChangelogLayout = () => (
```
Comment (rosendi) [inline src/routes/changelog/index.js]: Technically it's still `ChangelogRoutes`

**Comment by rosendi** [comment]
@radiansz I don't see any issues expect a few comments.

**Comment by radiansz** [comment]
Fixed

**Comment by rosendi** [comment]
@radiansz Blocked till #429 isn't merged. Check the issue with images there. am I right that they should be available at '/img/...'?

**Comment by zouxuoz** [comment]
lgtm

**Comment by rosendi** [comment]
lgtm

**Pull request merged**